lightningd/connect_control.c

```diff
@@ -177,7 +177,13 @@ static void json_connect(struct command *cmd,
 			}
 			addr.port = port;
 		} else {
-			addr.port = DEFAULT_PORT;
+			/* BOLT #1:
+			 *
+			 * The default TCP port is 9735. This corresponds to
+			 * hexadecimal `0x2607`, the Unicode code point for
+			 * LIGHTNING.
+			 */
+			addr.port = 9735;
```

Comment:
Why this in particular for outgoing, and not use default?

Reply:
I deleted DEFAULT_PORT for clarity, and to catch anyone assuming it.